src/data/markdown/docs/02 javascript api/07 k6-experimental/01 browser.md

```diff
@@ -86,9 +86,17 @@ $ k6 run script.js
 ```
 
 ```bash
-# When using the `k6:master-with-browser` Docker image, you need to add `--cap-add=SYS_ADMIN`
-# to grant further system permissions on the host for the Docker container.
-docker run --rm -i --cap-add=SYS_ADMIN grafana/k6:master-with-browser run - <script.js
+# WARNING!
+# Be aware that grafana/k6:master-with-browser launches Chrome browser setting 'no-sandbox'
+# argument. Use it only with trustworthy websites.
```

Comment:
```suggestion
# Be aware that grafana/k6:master-with-browser launches Chrome browser by setting the
# 'no-sandbox' argument. Use it only with trustworthy websites.
```

Reply:
Changed in https://github.com/grafana/k6-docs/pull/1330/commits/99fcfbb5689ec4c1cf741fd8353e9a9aacdcaf57.